src/System.Management.Automation/engine/lang/scriptblock.cs

```diff
@@ -103,15 +103,10 @@ internal static ScriptBlock Create(ExecutionContext context, string script)
         /// context is provided.
         /// </summary>
         /// <param name="script">The string to compile.</param>
-        public static ScriptBlock Create(string script)
-        {
-            return Create(new Language.Parser(), null, script);
-        }
+        public static ScriptBlock Create(string script) => Create(new Language.Parser(), null, script);
```

Comment:
If we do style formatting it would be nice to have named parameters.
 Below too.

Reply:
That's a great idea! I'll go through and add them, thanks!